tasks/tests.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from invoke import run, task
+
+from . import copy
+from . import requirements
+from . import git_tasks
+
+
+@task(requirements.install, git_tasks.clone_st2_repo, copy.copy_pack_to_subdirectory)
+def packs_resource_register(ctx):
+    print("")
+    print("==================== packs-resource-register ====================")
+    print("")
+    base_branch = os.environ.get('BASE_BRANCH', 'origin/master')
+    files = []
+    top_level_git_dir = ctx.run('git rev-parse --show-toplevel').stdout.splitlines()[0]
+    with ctx.cd(top_level_git_dir):
+        files = ctx.run("git diff --relative --diff-filter=ACMRTUXB "
+                        "--name-only {}".format(base_branch)).stdout.splitlines()
+    if files:
+        ctx.run("st2-check-print-pack-tests-coverage {pack_dir}".format(pack_dir=top_level_git_dir))
```

Comment:
I assume this should be calling a different script which performs the resource register check?

Reply:
Good catch, will fix.